### ISSUE TITLE
Avhievement Checklist not working

### ISSUE THREAD

**Issue by isblagi** [Annosz/EU4AchievementChecklist#1]
First of all, than you for the achievements checklist page. It's the best way to track my EU4 achievements by far.

As the title says, when I click the link to the page it doesn't work. The browser opens a new tab that gives me this error:
"Application error
An error occurred in the application and your page could not be served. If you are the application owner, [check your logs for details](https://devcenter.heroku.com/articles/logging#view-logs). You can do this from the Heroku CLI with the command
heroku logs --tail"

Hope you can fix it because as I said your solution is the best, easiest and more comfortable way to keep achievements under control. A lot of thanks in advance! :)

**Comment by Annosz** [comment]
Hi! Thanks, I did not even know that this website is used by anyone but me :) I always want to finish it up and share it, but somehow I never got around to do that.

I've checked and the other deployment is still working: [https://eu4achievementchecklist.azurewebsites.net](https://eu4achievementchecklist.azurewebsites.net)
I want to use this in the long run, so I will not fix the one ending with .herukoapp.com. I've changed the description too on the Github page, so now it will link to the working one.
